qa/tasks/ceph_manager.py

```diff
@@ -1055,7 +1056,7 @@ def wrapper(self):
                 raise
         return wrapper
 
-    @log_exc
+    @OSDThrasher.log_exc
```

Comment:
```
2020-01-24T10:31:28.462 ERROR:teuthology.run_tasks:Saw exception from tasks.
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 86, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 64, in run_one_task
    task = get_task(taskname)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 28, in get_task
    module = _import('tasks', module_name, task_name, fail_on_import_error=True)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 52, in _import
    0,
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/virtualenv/local/lib/python2.7/site-packages/gevent/builtins.py", line 96, in __import__
    result = _import(*args, **kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-01-23-2212/qa/tasks/ceph.py", line 21, in <module>
    from ceph_manager import CephManager, write_conf
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/virtualenv/local/lib/python2.7/site-packages/gevent/builtins.py", line 96, in __import__
    result = _import(*args, **kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-01-23-2212/qa/tasks/ceph_manager.py", line 123, in <module>
    class OSDThrasher(Thrasher):
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-01-23-2212/qa/tasks/ceph_manager.py", line 1066, in OSDThrasher
    @OSDThrasher.log_exc
NameError: name 'OSDThrasher' is not defined
```

http://pulpito.ceph.com/kchai-2020-01-24_02:23:44-rados-wip-kefu-testing-2020-01-23-2212-distro-basic-smithi/4698061/

Reply:
I moved the `log_exc` method to top level again. @tchaikov Could you retest please?